network-api/networkapi/wagtailpages/tests/research_hub/test_author_index.py

```diff
@@ -150,16 +150,16 @@ def test_get_author_detail_context_multiple_detail_pages(self):
             author_profile=self.research_profile,
         )
 
-        # author, locale, detail pages.
+        # locale & detail pages.
         with self.assertNumQueries(2):
-            context = self.author_index.get_latest_research(
+            latest_research = self.author_index.get_latest_research(
                 author_profile=self.research_profile,
             )
-            self.assertEqual(len(context), 3)
-            self.assertIn(detail_page_1, context)
-            self.assertIn(detail_page_2, context)
-            self.assertIn(detail_page_3, context)
-            self.assertNotIn(self.detail_page, context)
+            self.assertEqual(len(latest_research), 3)
+            self.assertIn(detail_page_1, latest_research)
+            self.assertIn(detail_page_2, latest_research)
+            self.assertIn(detail_page_3, latest_research)
+            self.assertNotIn(self.detail_page, latest_research)
```

Comment:
Sorry I missed this before, but the test method should be renamed to reflect what is tested. I guess `test_get_latest_research` should work.